K2Bridge/Models/Response/Aggregations/ExtendedStats/ExtendedStatsAggregate.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// See LICENSE file in the project root for full license information.
+
+namespace K2Bridge.Models.Response.Aggregations
+{
+    using Newtonsoft.Json;
+
+    /// <summary>
+    /// Describes extended stats response element for standard deviation.
+    /// </summary>
+    public class ExtendedStatsAggregate : IAggregate
```

Comment:
Our class is very barren compared to the one from kibana:
```json
{
  ...

  "aggregations": {
    "grades_stats": {
      "count": 2,
      "min": 50.0,
      "max": 100.0,
      "avg": 75.0,
      "sum": 150.0,
      "sum_of_squares": 12500.0,
      "variance": 625.0,
      "variance_population": 625.0,
      "variance_sampling": 1250.0,
      "std_deviation": 25.0,
      "std_deviation_population": 25.0,
      "std_deviation_sampling": 35.35533905932738,
      "std_deviation_bounds": {
        "upper": 125.0,
        "lower": 25.0,
        "upper_population": 125.0,
        "lower_population": 25.0,
        "upper_sampling": 145.71067811865476,
        "lower_sampling": 4.289321881345245
      }
    }
  }
}
```
Do you know if we need the other fields?
Should we add them just for parity?

Reply:
We did not find any cases where the population and sampling values are used.

Reply:
Ok, but maybe leave a comment or something that we are aware that we don't have full parity to kibana

Reply:
i'm working on it. I've found how to retrieve these values from ADX

Reply:
Done

Reply:
Great!